### PR TITLE
Added Previous events page & Updated the events

### PR DESCRIPTION
**Pull request by Sc0rpi0n101**
* Added a new Previous Events page and data file
* Removed VUI event from upcoming events and added it to the previous events

image

image

### CODE REVIEW

**Comment by Sc0rpi0n101** [comment]
@vipulgupta2048

File: data/prev_events.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "title": "Introduction to VUI",
+    "date": "9th January",
+    "poster": "images/flashbox/vui_event_poster.png",
+    "startTime": "02:30 PM",
+    "endTime": "04:40 PM",
+    "description": "In this beginner friendly event learn what is VUI, how it functions and create your own VUI app using Alexa and Google Assistant without writing even a single line of code."
+  }
+]
```
Comment (vipulgupta2048) [inline data/prev_events.json]: Wouldn't previous events include all previous events or are we considering only 2020. 

IMO it should extend to at least 2019

Comment (Sc0rpi0n101) [reply data/prev_events.json]: I also think all the events should be added. But I had no information regarding the events.

I can add more now. Where can I find the previous events?

Comment (vipulgupta2048) [reply data/prev_events.json]: Previous events data can be found in previous commits

Comment (vipulgupta2048) [reply data/prev_events.json]: or file history

**Comment by Sc0rpi0n101** [comment]
ping @vipulgupta2048

**Pull request merged**